include/glow/Base/Tensor.h

```diff
@@ -300,8 +300,16 @@ class Tensor final {
              "Offsets must match.");
       return isEqualImpl<int8_t>(other, allowedError);
     case ElemKind::Int16QTy:
+      assert(getType().getScale() == other.getType().getScale() &&
```

Comment:
Comparison needs to be fixed properly for all quantized types, see https://github.com/pytorch/glow/issues/2206